introspection/introspection.go

```diff
@@ -0,0 +1,81 @@
+package introspection
+
+import (
+	"bytes"
+	_ "embed"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+
+	"github.com/wundergraph/graphql-go-tools/pkg/astprinter"
+	wunderintro "github.com/wundergraph/graphql-go-tools/pkg/introspection"
+)
+
+//go:embed introspection.gql
+var introspectionQuery string
+
+type jsonBody struct {
+	Data json.RawMessage `json:"data"`
+}
+
+func Load(url string) ([]byte, error) {
+	jsonSchema, err := fetch(url)
+	if err != nil {
+		return []byte{}, err
+	}
+	schema, err := jsonToSDL(jsonSchema)
+	if err != nil {
+		return []byte{}, err
+	}
+
+	return schema, nil
+}
+
+// Schema will fetch a schema at the given url and unwrap the response and
```

Comment:
```suggestion
// fetch will fetch a schema at the given url and unwrap the response and
```

right?

Reply:
Fixed, we should be documenting the public function, not this private function 🙈